lib/membrane_wav/parser.ex

```diff
@@ -3,7 +3,7 @@ defmodule Membrane.WAV.Parser do
   Element responsible for parsing WAV files.
 
   It requires WAV file in uncompressed, PCM format on the input (otherwise error is raised) and
-  provides raw audio on the output. WAV header is parsed to extract metadata for creating caps.
+  provides raw audio on the output. WAV header is parsed to extract metadata for creating formats.
```

Comment:
the plural form doesn't fit here plus the word 'creating' seems a bit off without caps, Id' rephrase it like this:
```suggestion
  provides raw audio on the output. WAV header is parsed to extract metadata of the raw audio format.
```